### PR TITLE
Improve doc, pass value in VideoMessage if is gif and let the user decide the wpp client version

### PR DESCRIPTION
**Pull request by houstondapaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,13 +54,21 @@ func (myHandler) HandleVideoMessage(message whatsapp.VideoMessage) {
...
+}
+
 func (myHandler) HandleJsonMessage(message string) {
 	fmt.Println(message)
 }
 
 wac.AddHandler(myHandler{})
 ```
-The message handlers are all optional, you don't need to implement anything but the error handler to implement the interface. The ImageMessage and VideoMessage provide a Download function to get the media data.
+The message handlers are all optional, you don't need to implement anything but the error handler to implement the interface. The ImageMessage, VideoMessage, AudioMessage and AudioMessage provide a Download function to get the media data.
```
Comment (SchulteMK) [inline README.md]: AudioMessage appears twice

Comment (houstondapaz) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -50,17 +50,25 @@ func (myHandler) HandleImageMessage(message whatsapp.ImageMessage) {
 	fmt.Println(message)
 }
 
+func (myHandler) HandleDocumentMessage(message whatsapp.VideoMessage) {
```
Comment (SchulteMK) [inline README.md]: Thank you for contributing to the project! 👍 
another small mistake was made: HandleDocumentMessage takes a DocumentMessage as parameter and not VideoMessage.

Comment (houstondapaz) [reply README.md]: > After that small fix I will directly merge the branch. Sorry that I did not spot that yesterday alreasy

No problem, my bad, I did not pay attention at the details

**Pull request merged**